USING_DOCKER.md

```diff
@@ -40,16 +40,25 @@ cd <top-level-PRQL-directory>
 docker build -t prql .
 ```
 
-_Optional:_ To guard against silly mistakes before each `git commit ...`,
-install the `pre-commit` program
-on the machine that hosts your Docker container.
-Then (one time) run `pre-commit install-hooks` to configure your
-`git` client with the contents of the `.pre-commit-config.yaml` file. [^1]
+_Optional:_ The `pre-commit` program makes a quick check
+for silly mistakes of all files: rust formatting,
+`.yaml` problems, Markdown style, pretty-printing, etc.
```

Comment:
I do think it's worth linking to https://github.com/prql/prql/blob/main/DEVELOPMENT.md#tests here; we give an explanation of what they do and how they fit into the overall testing approach. Or do you think that's kinda confusing? Please criticize away for that doc!

Reply:
Definitely. DRY (Don't Repeat Yourself)